crypto/.js/src/main/scala/org/bitcoins/crypto/BCryptoCryptoRuntime.scala

```diff
@@ -148,49 +151,41 @@ trait BCryptoCryptoRuntime extends CryptoRuntime {
 
   override def tweakMultiply(
       publicKey: ECPublicKey,
-      tweak: FieldElement): ECPublicKey = ???
-
-  override def add(pk1: ECPrivateKey, pk2: ECPrivateKey): ECPrivateKey = ???
-
-  override def add(bytes: ByteVector, pk2: ECPrivateKey): ByteVector = ???
+      tweak: FieldElement): ECPublicKey = {
+    val keyBuffer = ecdsa.publicKeyTweakMul(publicKey.bytes, tweak.bytes, true)
+    ECPublicKey.fromBytes(keyBuffer)
+  }
 
-  override def add(pk1: ECPublicKey, pk2: ECPublicKey): ECPublicKey = ???
+  override def add(pk1: ECPublicKey, pk2: ECPublicKey): ECPublicKey = {
+    try {
+      val keyBuffer =
+        ecdsa.publicKeyCombine(js.Array(pk1.bytes, pk2.bytes), true)
+      ECPublicKey.fromBytes(keyBuffer)
+    } catch {
+      case ex: JavaScriptException =>
+        // check for infinity
+        if (
+          pk1.isCompressed && pk2.isCompressed &&
+          ((pk1.bytes.head == 0x02 && pk2.bytes.head == 0x03) || (pk1.bytes.head == 0x03 && pk2.bytes.head == 0x02)) &&
```

Comment:
if the pub keys are not compressed they should be compressed prior to this comparison, no reason to fail to add pubkeys together that aren't compressed